apps/catalogue/src/views/HarmonizationRow.vue

```diff
@@ -56,6 +57,7 @@ export default {
 
         const baseVariable = this.resourceMappings.find((mapping) => {
           return (
+            mapping.toVariable &&
```

Comment:
.?

Reply:
then you would return 'false' right instead of the desired output?